From e65f1cb0f385c7ae743f8c51a24c2e8c7cda9959 Mon Sep 17 00:00:00 2001
From: jgudec <jakov.gudec@gmail.com>
Date: Fri, 21 Apr 2023 16:49:15 +0100
Subject: [PATCH 2/2] SystemUI: styles: Change Keyguard style on light
 wallpapers

This fixes the horrendously dark lock screen clock color that DEFINITELY should not exist on a rom called Pixel Experience as it DEFINITELY doesn't look like that on Pixel devices.

Change-Id: I330143ec335661967df1cc3129b04791891a192f
---
 core/java/android/service/wallpaper/WallpaperService.java | 2 +-
 packages/SystemUI/res/values/styles.xml                   | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/core/java/android/service/wallpaper/WallpaperService.java b/core/java/android/service/wallpaper/WallpaperService.java
index 2d1a41e92a99..017aa5fd6769 100644
--- a/core/java/android/service/wallpaper/WallpaperService.java
+++ b/core/java/android/service/wallpaper/WallpaperService.java
@@ -285,7 +285,7 @@ public abstract class WallpaperService extends Service {
         private Context mDisplayContext;
         private int mDisplayState;
         private @Surface.Rotation int mDisplayInstallOrientation;
-        private float mWallpaperDimAmount = 0.05f;
+        private float mWallpaperDimAmount = 0.45f;
         private float mPreviousWallpaperDimAmount = mWallpaperDimAmount;
         private float mDefaultDimAmount = mWallpaperDimAmount;
 
diff --git a/packages/SystemUI/res/values/styles.xml b/packages/SystemUI/res/values/styles.xml
index b11b6d633f14..e726722566a4 100644
--- a/packages/SystemUI/res/values/styles.xml
+++ b/packages/SystemUI/res/values/styles.xml
@@ -321,11 +321,11 @@
     </style>
 
     <style name="Theme.SystemUI.LightWallpaper">
-        <item name="wallpaperTextColor">@*android:color/primary_text_material_light</item>
+        <!-- <item name="wallpaperTextColor">@*android:color/primary_text_material_light</item>
         <item name="wallpaperTextColorSecondary">@*android:color/secondary_text_material_light</item>
         <item name="wallpaperTextColorAccent">@color/material_dynamic_secondary40</item>
         <item name="android:colorError">@*android:color/error_color_material_light</item>
-        <item name="shadowRadius">0</item>
+        <item name="shadowRadius">0</item> -->
 
         <!-- Needed for MediaRoute chooser dialog -->
         <item name="*android:isLightTheme">true</item>
-- 
2.17.1

